mleap-spark/src/test/scala/org/apache/spark/ml/parity/regression/AFTSurvivalRegressionParitySpec.scala

```diff
@@ -26,4 +26,6 @@ class AFTSurvivalRegressionParitySpec extends SparkParityBase {
       setLabelCol("loan_amount").
       setQuantilesCol("quant").
       setPredictionCol("prediction"))).fit(dataset)
+
+  override val unserializedParams = Set("featuresCol", "labelCol", "quantilesCol", "predictionCol", "stringOrderType", "maxIter", "tol")
```

Comment:
"featuresCol", "quantilesCol", "predictionCol" are actually set, so they can be removed from unserializedParams.